packages.json

```
@@ -66,5 +66,9 @@
     {
         "name": "guardrail-azure-content-safety-content-moderation",
         "version": "0.0.1"
+    },
+    {
+        "name": "guardrail-semantic-cache",
+        "version": "1.0.0"
```

```suggestion
        "version": "0.0.1"
```
since there should have a diff between the version in this json file and ballerina.toml.

---

Changed version to 0.0.1 as suggested